src/components/Checkbox/Checkbox.tsx

```diff
@@ -81,6 +96,11 @@ const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
         {labelText && showLabel && (
           <Label htmlFor={checkboxID}>{labelText}</Label>
         )}
+        {(helperText || (errored && errorText)) && showLabel && (
```

Comment:
Similar comment to the other PR, I think you can replace `(helperText || (errored && errorText))` with `footnote`.

Reply:
Yes, this change worked well.